org.lflang/src/org/lflang/generator/EclipseErrorReporter.java

```diff
@@ -173,6 +174,16 @@ public String reportWarning(String message) {
         return report(message, Severity.WARNING, null, null);
     }
 
+    /**
+     * Report an info.
```

Comment:
Since this is an override, you could just use:

```
/**
 * {@inheritdoc}
 */
```
(Or leave out the comment entirely, because I think JavaDoc is smart enough to inherit docs from the interface or super class automatically).

Reply:
comment is removed.